engine/src/main/resources/org/camunda/bpm/engine/db/upgrade/cockroachdb_engine_7.20_to_7.21.sql

```diff
@@ -17,3 +17,6 @@
 
 insert into ACT_GE_SCHEMA_LOG
 values ('1000', CURRENT_TIMESTAMP, '7.21.0');
+
+alter table ACT_RU_EXT_TASK
```

Comment:
❌For CockroachDB, schema changes must come before table writes. Check [7.15 migration script](https://github.com/camunda/camunda-bpm-platform/blob/60f9d233b6747624c8957194dae2d119d3b638eb/engine/src/main/resources/org/camunda/bpm/engine/db/upgrade/cockroachdb_engine_7.14_to_7.15.sql) for example.